python/vpc-ec2-local-zones/vpc_ec2_local_zones/vpc_ec2_local_zones_stack.py

```diff
@@ -0,0 +1,182 @@
+        wp_as.connections.allow_from(alb, ec2.Port.tcp(8080), "ALB access on target instance")
+        return alb
+
+
+    # method to create a new EC2 instance with MySQL installed through User Data script
+    # see user_data/db_mysql txt file to see the User Data script
+    def create_db_mysql(self, vpc):
+        amzn_linux = ec2.MachineImage.latest_amazon_linux(generation=ec2.AmazonLinuxGeneration.AMAZON_LINUX_2)
+        user_data = self.get_user_data("db_mysql")
+        db = ec2.Instance(self, "DBInLZ",
+                 vpc=vpc,
+                 instance_type=ec2.InstanceType.of(ec2.InstanceClass.T3, ec2.InstanceSize.MEDIUM),
+                 machine_image=amzn_linux,
+                 user_data=ec2.UserData.custom(user_data),
+                 vpc_subnets=ec2.SubnetSelection(availability_zones=[LZ_NAME],subnet_type=ec2.SubnetType.PRIVATE_ISOLATED),
+                )
+        return db
+
+    # method to create a new AutoScaling group for managing WordPress instances.
+    # see user_data/wp_webserver txt file to see the User Data script
+    def create_wp_webserver(self, vpc, db):
+        amzn_linux = ec2.MachineImage.latest_amazon_linux(generation=ec2.AmazonLinuxGeneration.AMAZON_LINUX_2)
+        user_data = self.get_user_data("wp_webserver")
+        # we need to modify the user data script to provide the dns name of the
+        # mysql database installed in the private subnet.
+        user_data = re.sub('dbhost', db.instance_private_dns_name, user_data)
+
+        wp_as = autoscaling.AutoScalingGroup(self,
+                "WordPressAS",
+                vpc=vpc,
+                instance_type=ec2.InstanceType.of(ec2.InstanceClass.T3, ec2.InstanceSize.MEDIUM),
+                machine_image=amzn_linux,
+                user_data=ec2.UserData.custom(user_data),
+                vpc_subnets=ec2.SubnetSelection(availability_zones=[LZ_NAME],
+                                subnet_type=ec2.SubnetType.PRIVATE_ISOLATED),
+        )
+        # allow connections from wp_instance to db
+        db.connections.allow_from(wp_as, ec2.Port.tcp(3306), "WP access on db port")
+        return wp_as
+
+    def get_user_data(self, filename):
+        with open('./user_data/' + filename) as f:
+            user_data = f.read()
+        return user_data
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # create new VPC in the Local Zone
+        vpc = self.create_VPC()
+
+        # create NAT instance
+        nat_instance = self.create_nat_instance(vpc)
+        self.add_route_to_nat(vpc, nat_instance)
+
+        # create db mysql
+        db = self.create_db_mysql(vpc)
+        # create an AutoScaling groups to manage web server with WordPress
+        wp_as = self.create_wp_webserver(vpc, db)
+        # create ALB
+        alb = self.create_ALB_in_lz(vpc, wp_as)
+        CfnOutput(self, "ALB DNS name: ", value=alb.load_balancer_dns_name)
```

Comment:
```suggestion
        CfnOutput(self, "ALB DNS name: ", value=alb.load_balancer_dns_name)
        CfnOutput(self, "URL: ", value='http://'+alb.load_balancer_dns_name)
```

How about adding an extra URL output like this so customer can simply click and open the browser?

Reply:
I like it!